storage/connect/plugutil.cpp

```diff
@@ -426,7 +426,7 @@ char *PlugGetMessage(PGLOBAL g, int mid)
   if (n == 0) {
     DWORD rc = GetLastError();
     msg = (char*)PlugSubAlloc(g, NULL, 512);   // Extend buf allocation
-    n = sprintf(msg, "Message %d, rc=%d: ", mid, rc);
+    n = snprintf(msg, 512, "Message %d, rc=%d: ", mid, rc);
```

Comment:
Please make 512 a `#define` and use it on the lines above and below this so that a change later won't break this.

Reply:
> Please make 512 a `#define` and use it on the lines above and below this so that a change later won't break this.

Changed 512 to const int variable